src/main/java/org/folio/rest/impl/BLUsersAPI.java

```diff
@@ -914,4 +931,163 @@ else if(include.get(i).equals(SERVICEPOINTS_INCLUDE)) {
     });
   }
 
+  /**
+   *
+   * @param locateUserFields - a list of fields to be used for search
+   * @param value - a value to search
+   * @return
+   */
+  private String buildQuery(List<String> locateUserFields, String value) {
+    return locateUserFields.stream()
+      .map(field -> new StringBuilder(field).append("==\"").append(value).append("\"").toString())
```

Comment:
`field` and `value` needs to be quoted/escaped if it contains whitespace or any of there characters: ( ) = <> " /
See https://www.loc.gov/standards/sru/cql/spec.html#bnf

Reply:
The **field** could be just a user entity field name so it can't contain chars to escape. The **value** is already quoted. Or it should be quoted in another way. Could you provide an example for this case please. Thank you very much.